402.md

```diff
@@ -0,0 +1,71 @@
+# NIP-402
+
+## Classifieds
+
+`draft` `optional` `author:erskingardner`
+
+This NIP defines `kind:30402`: a parameterized replaceable event to describe classified listings that list any arbitrary product, service, or other thing for sale or offer and includes enough structured metadata to make them useful.
+
+The category of classifieds includes a very broad range of physical goods, services, work opportunities, rentals, free giveaways, personals, etc. and is distinct from the more strictly structured marketplaces defined in [NIP-15](https://github.com/nostr-protocol/nips/blob/master/15.md) that often sell many units of specific products through very specific channels.
+
+The structure of these events is very similar to [NIP-23](https://github.com/nostr-protocol/nips/blob/master/23.md) long-form content events.
+
+### Draft / Inactive Listings
+
+`kind:30403` has the same structure as `kind:30402` and is used to save draft or inactive classified listings.
+
+### Content
+
+The `.content` field should be a description of what is being offered and by whom. These events should be a string in Markdown syntax.
+
+### Author
+
+The `.pubkey` field of these events are treated as the party creating the listing.
+
+### Metadata
+
+- For "tags"/"hashtags" (i.e. categories or keywords of relevance for the listing) the `"t"` event tag should be used, as per [NIP-12](https://github.com/nostr-protocol/nips/blob/master/12.md).
+
+The following tags, used for structured metadata, are standardized and SHOULD be included. Other tags may be added as necessary.
+
+- `"title"`, a title for the listing
+- `"summary"`, for short tagline or summary for the listing
```

Comment:
I think it’s a good idea to have an abstract to digest the information of the market item proposed for sale, however the distinction between “title” and “summary” is unclear, is the “summary” supposed to be a list of product/service attributes (quantity of items, quality, basket of heterogenous goods sell as “one”) ?